docs/_extra/api-reference/schemas/annotation-create.yaml

```diff
@@ -0,0 +1,79 @@
+Annotation:
+  type: object
+  required:
+    - uri
+  properties:
+    uri:
+      type: string
+      format: uri
+      description: The URI to the annotation's target
+
+    document:
+      type: object
+      description: Further metadata about the target document
+      properties:
```

Comment:
Looking at the code which handles this field (see [h/document_claims.py](https://github.com/hypothesis/h/blob/2d6e98e57f1f1b02f3e672ff9583859bc6f4dc37/h/util/document_claims.py#L81)) the "document" object can contain arbitrarily-named dicts. The [`DocumentMeta` class](https://github.com/hypothesis/client/blob/bbd65498e3892df701af07862e2511f6d9602df3/src/annotator/plugin/document.js#L24) in the client sets some other ones besides `dc` and `highwire` as well.

I'm not sure we should document the fact that we support arbitrary properties though, just documenting some of the most commonly used ones as done here may make more sense.

The one important one which is missing I think is the `title` field, which gets used on activity pages and by the bouncer service.